extensions/astro/languages/astro/highlights.scm

```diff
@@ -22,4 +22,4 @@
   ">"
   "</"
   "/>"
-] @tag.delimiter
+] @punctuation.bracket
```

Comment:
Would it be better to use `punctuation.tag` or `punctuation.bracket.tag` or something like that?